<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetaDataWidget</class>
 <widget class="QWidget" name="MetaDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="upperArea" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="personsLayout">
        <item>
         <widget class="QLabel" name="participantLabel">
          <property name="text">
           <string>Name, Vorname</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="participantLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="instructorLabel">
          <property name="text">
           <string>Untersucher(in)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="instructorLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="datesLayout">
        <item>
         <widget class="QLabel" name="dateOfBirthLabel">
          <property name="text">
           <string>Geburtsdatum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateOfBirthDateEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="dateOfTestLabel">
          <property name="text">
           <string>Untersuchungsdatum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateOfTestDateEdit">
          <property name="displayFormat">
           <string>M/d/yyyy</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="ageLabel">
          <property name="text">
           <string>Alter am Testtag</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ageLineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="remarksLabel">
     <property name="text">
      <string>Bemerkungen</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="remarksPlainTextEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
